project_event/models/project_project.py

```diff
@@ -85,9 +85,30 @@ def action_cancel(self):
     def action_accept(self):
         if self.state in ['draft', 'option', 'postponed', 'canceled']:
             self.send_message('accepted')
+            res = ''
```

Comment:
This piece of code is very similar to action_option, the only difference I see is on the message. Don't you think we could create a new method and call it with different messages?